app/controllers/manage_consents_controller.rb

```diff
@@ -83,16 +83,16 @@ def current_step
   def finish_wizard_path
     success_flash_after_patient_update(
       patient: @patient,
-      view_record_link: session_patient_triage_path(@session, @patient)
+      view_record_link: session_patient_path(@session, id: @patient.id)
     )
 
-    case @route
+    case @section
     when "consents"
-      consents_session_path(@session)
+      session_consents_path(@session)
     when "triage"
       triage_session_path(@session)
     else
-      vaccinations_session_path(@session)
+      session_vaccinations_path(@session)
```

Comment:
- session_consents_path
- triage_session_path
- session_vaccinations_path

Triage is the odd one out, how come? I thought it should be `session_triage_path` according to the new routes